bin/xud

```diff
@@ -53,6 +53,18 @@ const { argv } = require('yargs')
       describe: 'Data directory for lnd',
       type: 'string',
     },
+    'lnd.certificate': {
+      describe: 'Certificate for lnd',
+      type: 'string',
+    },
+    'lnd.host': {
+      describe: 'Host of the lnd gRPC interface',
+      type: 'string',
+    },
+    'lnd.macaroon': {
+      describe: 'Macaroon file for lnd',
```

Comment:
Likewise, this can be `macaroonpath` and the description can say it's the "Path to the admin macaroon for lnd."

Reply:
Do we really the admin macaroon? I think the only reason you are using it instead of the invoice macaroon is that you are using the `getinfo` method of LND? And it is just a placeholder for things to come later i suppose.